doc/user/content/sql/namespaces.md

```diff
@@ -9,16 +9,62 @@ menu:
     weight: 126
 ---
 
-Materialize mimics SQL standard's namespace hierarchy, which is:
+Materialize follows SQL standard's namespace hierarchy for most objects. The
+Materialize structure is:
 
 - Databases (highest level)
 - Schemas
-- Tables, views, sources
+- Tables
+- Sources
+- Views
+- Materialized views
+- Sinks
+- Indexes
+- Types
+- Functions
+- Secrets
```

Comment:
This sort of suggests tables are a level above sources which are a level above views, etc. We might want to make it more obvious which objects are on the same level.